Gpt 88 250 k scanned geological maps

oehm-smith opened the pull request:
Code mod to present multiple features in tabs - similar to the way AuScope Portal presents query results for mineral occurrences.  Needs to be activated in the portal.  I'd created a pull request for this which I closed as incomplete.  See it for a description of the work - https://github.com/AuScope/portal-core/pull/81.

victortey commented:
Example on how this can be used can be found on 

https://github.com/GeoscienceAustralia/geoscience-portal/pull/80/files

It would be great if we can add this as a comment in the WMSMultipleTabDisplayQuerier.js file so that anyone that would like to use this can easily refer to the code on how to implement this. I will continue to merge this in.

victortey commented:
I have updated the file and added the comment in. No further work required.

oehm-smith commented:
Great thanks!

From: Victor [mailto:notifications@github.com]
Sent: Wednesday, 6 January 2016 12:33 PM
To: AuScope/portal-core
Cc: Smith Brooke
Subject: Re: [portal-core] Gpt 88 250 k scanned geological maps (#91)


I have updated the file and added the comment in. No further work required.

—
Reply to this email directly or view it on GitHub<https://github.com/AuScope/portal-core/pull/91#issuecomment-169190740>.

Geoscience Australia Disclaimer: This e-mail (and files transmitted with it) is intended only for the person or entity to which it is addressed. If you are not the intended recipient, then you have received this e-mail by mistake and any use, dissemination, forwarding, printing or copying of this e-mail and its file attachments is prohibited. The security of emails transmitted cannot be guaranteed; by forwarding or replying to this email, you acknowledge and accept these risks.
-------------------------------------------------------------------------------------------------------------------------
